beebop/utils.py

```diff
@@ -94,3 +94,29 @@ def replace_filehashes(folder, filename_dict):
                 if f_key in line:
                     line = line.replace(f_key, f_value)
             print(line)
+
+
+def add_query_ref_status(fs, p_hash, filename_dict):
+    ET.register_namespace('', "http://graphml.graphdrawing.org/xmlns")
+    ET.register_namespace('xsi', "http://www.w3.org/2001/XMLSchema-instance")
```

Comment:
should these registrations be done at the top-level? I am not familiar with the package at all